docs/atomic-uninstall.1.md

```diff
@@ -49,5 +50,8 @@ Any additional arguments will be appended to the command.
 **-n** **--name**=""
    If name is specified `atomic uninstall` will uninstall the named container from the system, otherwise it will uninstall the container images.
 
+**--system**
+   Uninstall a system container.
```

Comment:
I thought these are two different namespaces, in theory it is possible to have a Docker container and a system container with the same name.  Should we not allow to install the system container if there is already a docker container with that name?  Though the user can still create the Docker container in another way after a system container with the same name is running.  How then it will be possible to remove only the system container with `atomic` (or viceversa, the Docker container)?